cmd/mesh/manifest.go

```diff
@@ -89,6 +93,27 @@ func genManifests(inFilename string, setOverlayYAML string) (name.ManifestMap, e
 	if err != nil {
 		return nil, err
 	}
+	// for installation path contains version number only
+	isp := mergedICPS.InstallPackagePath
+	if version.IsVersionString(isp) {
+		mergedICPS.InstallPackagePath = fmt.Sprintf(helm.InstallationPathTemplate, isp, isp)
```

Comment:
it would be more readable to write this as mergedICPS.InstallPackagePath = helm.InstallURLFromVersion(mergedICPS.version)
then, make InstallationPathTemplate private and InstallURLFromVersion should only need version string, not whole path.

Reply:
done